### PR TITLE
Create session-b.md

### PR DESCRIPTION
**Pull request by uditvira**
Proposing "steamlink" talk + workshop

### CODE REVIEW

**Comment by uditvira** [comment]
@dcwalk left the workshop description a little vague for now while I figure out content

File: sessions/session-b.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Session B: Creating low-power sensor networks using LoRa
+
+## Description
+
+The last few year have been an exciting time for the DIY electronics community. There has been an explosion of sensors, actuators, and micro-controllers to tinker with. Additionally, with access to inexpensive hardware like the Photon, Raspberry Pi, and ESP8266 it’s easier than ever to embed web-services directly in to our projects and this connectivty enables new applications. However, most of these projects connect over WiFi, which isn't well suited for low-power/low-bandwidth applications. WiFi usually limits our projects to indoors or tethered to our smartphones. These mobility and power limitations impede projects like distributed citizen science, connected guerrilla art, and remote sensing. LoRa is an inexpensive low-power/low-bandwidth radio technology that is one great solution to this problem. In urban settings LoRa radios can often provide low bandwidth connectivity of up to 1-2 kms and can be reasonably powered with AA batteries.
```
Comment (benhylau) [inline sessions/session-b.md]: - last few _years_
- _ESP8266,_
- directly _into_ our projects
- In urban _settings,_
- 1-2 _km_

File: sessions/session-b.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+In this session I will give a brief overview of LPWAN's (low-power wide area networks) and the technology behind LoRa. There is also a mini wokshop component to this session where we will set up infrastructure for an LPWAN to monitor sensors connect through LoRa radios.
```
Comment (benhylau) [inline sessions/session-b.md]: - In this _session,_

File: sessions/session-b.md (hunk 1 above)
Comment (dcwalk) [inline sessions/session-b.md]: Can you add links to a further description/resource RE: Photons, Pi's and ESP8266s?

File: sessions/session-b.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+* Hands-on demo of LoRa radios
+
+## Length
+
+50 mins
+
+## Materials
+
+* I have a few Adafruit LoRa feather boards (include LoRa radio + Arm M0 microcontroller board) for loan or purchase (each is about CAD $50)
+* Participants are encouraged to bring any arduinos or microcontroller projects they would like to add LoRa connectivity to
```
Comment (dcwalk) [inline sessions/session-b.md]: Do participants need a computer?
What materials do you as a presenter need that we (I) should make sure to have?

File: sessions/session-b.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Session B: Creating low-power sensor networks using LoRa
```
Comment (dcwalk) [inline sessions/session-b.md]: Is there a shorter title you could have, with a longer description after a colon (for linking on the website more easily?
e.g. _Creating sensor networks: Using LoRA to deploy low-power/low-bandwidth wide area networks_  (totally just an example, but something in that format?

File: sessions/session-b.md (hunk 1 above)
Comment (dcwalk) [reply sessions/session-b.md]: Oh also a link out to LoRA?

**Review by dcwalk — COMMENTED** [review]
Thanks @uditvira (looks awesome) ! Just a couple small additions/clarifications whenever you get the chance

**Comment by dcwalk** [comment]
Hey @uditvira I've updated the doc to correct the typos and add in links, the only outstanding bits:
- [x] short-form workshop title
- [x] clarifying presenter and participant materials (and the number? see session c: https://github.com/tomeshnet/mini-conf-2017/blob/master/sessions/session-c.md
- [x] confirming you are okay with updates.

Thanks!

**Comment by uditvira** [comment]
@dcwalk: sorry for the delay. Thanks for fixing it up. I've made some changes for the outstanding bits.

**Comment by dcwalk** [comment]
Thanks!

**Pull request merged**